fence-agents.spec.in

```diff
@@ -536,6 +537,20 @@ ping-heuristics.
 %{_sbindir}/fence_heuristics_ping
 %{_mandir}/man8/fence_heuristics_ping.8*
 
+%package heuristics-resource
+License: GPLv2+ and LGPLv2+
+Summary: Pseudo fence agent to affect other agents based on resource-heuristics
+Requires: pacemaker-cli
```

Comment:
You shouldnt depend on Pacemaker, as there are others using fence-agents, who might not need it, and we try to depend on as minimal as possible to not pull in a lot of stuff when installing -all.

Reply:
Thank you for your review. Just removed the dependency on pacemaker.
In addition, the default of crm_resource command path was changed to a variable of AC_PATH_PROG macro.

Reply:
Perfect. Thanks.

Reply:
With weak dependencies - where available - we might be able to address this issue in a more satisfactory way. fence_sbd for example is as well a candidate for thinking over this.

Reply:
Yeah. I guess we could do that, so users could do `yum install --suggests fence-agents-<name>`.

Reply:
Seems like --suggests doesnt work for install though.

Reply:
I was more thinking the other way round like e.g. requiring pacemaker by this one but fence-agents-all just suggesting this one.